stac_api/runtime/src/validation.py

```diff
@@ -33,24 +30,25 @@ async def dispatch(self, request: Request, call_next):
             try:
                 body = await request.body()
                 request_data = json.loads(body)
+
                 if re.match(
-                    f"^{path_prefix}/collections(?:/[^/]+)?$",
+                    "^.*?/collections(?:/[^/]+)?$",
```

Comment:
The original regex with path_prefix wasn't working with the tests. Do we need path_prefix here or can we just include wildcard? @slesaad